identity/verifier_fake.go

```diff
@@ -0,0 +1,9 @@
+package identity
+
+type VerifierFake struct{}
+
+func (verifier *VerifierFake) Verify(message []byte, signature Signature) bool {
+	signatureExpected := "signed" + string(message)
```

Comment:
This is coupled to fakeSigner, because signature generation is duplicated - changing one requires changing another. Maybe we can extract this somewhere?

Reply:
Moved to reusable hash function